cg/cli/workflow/fastq/utils.py

```diff
@@ -0,0 +1,7 @@
+from pathlib import Path
+
+import click
+
+
+def get_config_path(context: click.Context) -> str:
+    return Path(context.obj.data_delivery.base_path, "slurm_job_ids.yaml").as_posix()
```

Comment:
What do we need this path for? Nothing will be created there right?

Reply:
That path is wrong, but the config path is what we use to find the submitted jobs, right? Or is that via some other parameter? @Vince-janv

Reply:
Yes, we need a path to the the `slurm_job_ids.yaml` to track the rsync job. @beatrizsavinhas wants to be able to track the upload job for the fastq case in the new tab.

Reply:
The rsync dirs are named inconsistently, some use the database id and others the internal id. For example `271228_240108_10_53_05_784914` vs `betterweevil_240108_23_18_09_159755` which seems super annoying

Reply:
We will skip the upload job and fix separately for all pipelines and remove the rsync tab in trailblazer.